src/epics/timeline.js

```diff
@@ -0,0 +1,66 @@
+import { Observable } from 'rxjs';
+import { Map, OrderedMap } from 'immutable';
+import * as TimelineActions from 'app/actions/timeline';
+import RevisionEntity from 'app/entities/revision';
+
+const getWikiDomain = ( wiki, wikis ) => {
+	let wikiRecord = wikis.find( ( obj ) => {
+		return obj.get( 'id' ) === wiki;
+	} );
+
+	return wikiRecord.domain;
+};
+
+const buildUrl = ( store ) => {
+	const domain = getWikiDomain( store.getState().query.wiki, store.getState().wikis );
```

Comment:
Get rid of `getWikiDomain` and replace with `wikis.get( query.wiki )` because Immutable can find what you are looking for faster than you can. :) 
See https://facebook.github.io/immutable-js/docs/#/Map/get